src/Input/Token.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+/*
+ * This file is part of the PHP-CLI package.
+ *
+ * (c) Jitendra Adhikari <jiten.adhikary@gmail.com>
+ *     <https://github.com/adhocore>
+ *
+ * Licensed under MIT license.
+ */
+
+namespace Ahc\Cli\Input;
+
+use Ahc\Cli\Input\Option;
+
+use function \array_map;
```

Comment:
`\` not needed

Reply:
We can debate on that 😃 Anyway, I removed it.